includes/CLI/Plugin_Check_Command.php

```diff
@@ -103,8 +103,25 @@ public function __construct( Plugin_Context $plugin_context ) {
 	 * @throws Exception Throws exception.
 	 *
 	 * @SuppressWarnings(PHPMD.NPathComplexity)
+	 * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
 	 */
 	public function check( $args, $assoc_args ) {
+		/*
+		 * Bail early if the Plugin Checker is not activated.
+		 *
+		 * If the Plugin Checker is not activated, it will throw an error and
+		 * CLI commands won't be usable.
+		 */
+		if ( is_plugin_inactive( $this->plugin_context->basename() ) ) {
+			WP_CLI::error(
+				sprintf(
+					// translators: Plugin Checker slug.
+					__( '%s is not active.', 'plugin-check' ),
+					$this->plugin_context->basename()
+				)
```

Comment:
The translator comment must actually reference the placeholder, i.e. `// translators: %s: Plugin Checker slug.`

However, why not just simplify the error message?

```suggestion
				__( 'Plugin Checker is not active.', 'plugin-check' )
```